.github/workflows/deploy-queue.yml

```diff
@@ -83,6 +83,43 @@ jobs:
           
           echo "Linting and lockfile verification completed"
 
+  test:
```

Comment:
Nit: rename `test` to `test-integration` and `test-deploy-queue-action` to `test-e2e`